Makefile

```
@@ -93,22 +88,22 @@ COVER_ROOT                 := ./.coverage
 UNIT_COVER_PROFILE         := $(COVER_ROOT)/unit_coverprofile.out
 INTEGRATION_COVER_PROFILE  := $(COVER_ROOT)/integration_coverprofile.out
 DB_TOOL_COVER_PROFILE      := $(COVER_ROOT)/db_tool_coverprofile.out
-INTEG_COVER_PROFILE        := $(COVER_ROOT)/integ_$(PERSISTENCE_DRIVER)_coverprofile.out
-INTEG_XDC_COVER_PROFILE    := $(COVER_ROOT)/integ_xdc_$(PERSISTENCE_DRIVER)_coverprofile.out
-INTEG_NDC_COVER_PROFILE    := $(COVER_ROOT)/integ_ndc_$(PERSISTENCE_DRIVER)_coverprofile.out
+FUNCTIONAL_COVER_PROFILE   := $(COVER_ROOT)/functional_$(PERSISTENCE_DRIVER)_coverprofile.out
+FUNCTIONAL_XDC_COVER_PROFILE    := $(COVER_ROOT)/functional_xdc_$(PERSISTENCE_DRIVER)_coverprofile.out
+FUNCTIONAL_NDC_COVER_PROFILE    := $(COVER_ROOT)/functional_ndc_$(PERSISTENCE_DRIVER)_coverprofile.out
 SUMMARY_COVER_PROFILE      := $(COVER_ROOT)/summary.out
 
 # DB
 SQL_USER ?= temporal
 SQL_PASSWORD ?= temporal
 
-# Need the following option to have integration tests count towards coverage. godoc below:
+# Need the following option to have integration and functional tests count towards coverage. godoc below:
 # -coverpkg pkg1,pkg2,pkg3
 #   Apply coverage analysis in each test to the given list of packages.
 #   The default is for each test to analyze only the package being tested.
 #   Packages are specified as import paths.
-INTEG_TEST_COVERPKG := -coverpkg="$(MODULE_ROOT)/client/...,$(MODULE_ROOT)/common/...,$(MODULE_ROOT)/service/..."
-
+INTEG_TEST_COVERPKG := -coverpkg="$(MODULE_ROOT)/client/...,$(MODULE_ROOT)/common/...,$(MODULE_ROOT)/service/...,$(MODULE_ROOT)/temporal/..."
```

I guess this could be narrowed down to just:
```suggestion
INTEG_TEST_COVERPKG := -coverpkg="$(MODULE_ROOT)/common/persistence/..."
```

---

Updated